test-infra/src/main/java/com/datastax/oss/driver/api/testinfra/ccm/CcmBridge.java

```diff
@@ -409,7 +410,16 @@ protected void processLine(String line, int logLevel) {
       executor.setStreamHandler(streamHandler);
       executor.setWatchdog(watchDog);
 
-      int retValue = executor.execute(cli);
+      int retValue;
```

Comment:
I modified `CcmBridge` because the [latest commit ](https://github.com/apache/cassandra-java-driver/commit/7bc085bdfb337b91255573bc3e130815e280954a) on guava shaded broke the CI, cuz some how there are two source files that are not compiled in `mvn install` but compiled in `mvn verify`. Maybe we need to figure out why that happens and fix that instead, and revert the `CcmBridge` changes here.

Reply:
Can be reverted once #2000 is merged.